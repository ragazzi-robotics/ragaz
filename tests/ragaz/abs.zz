def main():

    print(abs(-123 as i8))
    print(abs(-123 as i16))
    print(abs(-123 as i32))
    print(abs(-123 as i64))
    print(abs(-123 as i128))
    print(abs(-123 as int))
    print(abs(-123 as f32))
    print(abs(-123 as f64))
    print(abs(-123 as float))
