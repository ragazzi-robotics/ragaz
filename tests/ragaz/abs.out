123
123
123
123
123
123
123.000000
123.000000
123.000000
